src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,49 @@
+package lotto.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Lotto {
+
+    private static final int START_RANGE = 1;
+    private static final int END_RANGE = 45;
+    private static final int COUNT = 6;
+    private final List<Integer> lottoNumbers;
+
+    public Lotto() {
+        this.lottoNumbers = generateRandomNumbers();
+    }
+
+    public Lotto(List<Integer> lottoNumbers) {
+        this.lottoNumbers = lottoNumbers;
+    }
+
+    private List<Integer> generateRandomNumbers() {
+        List<Integer> nums = IntStream.rangeClosed(START_RANGE, END_RANGE)// IntStream
+            .boxed() // Stream<Integer>
+            .collect(Collectors.toList()); // List<Integer>
```

Comment:
불필요한 주석을 제거해주세요! 😃
```suggestion
        List<Integer> nums = IntStream.rangeClosed(START_RANGE, END_RANGE)
            .boxed()
            .collect(Collectors.toList());
```

로또 게임에서는 아마도 수 많은 로또를 생성할거에요. 😃
지금 `generateRandomNumbers` 메서드를 보았을 때, 매번 1 ~ 45 까지의 숫자의 리스트를 생성한 뒤,  6자리의 숫자만 두고 리스트를 잘라내고 있어요. 🤔.  1부터 45까지의 로또 숫자들을 상수로 미리 만들어 두고 재사용하면 더 효율적으로 바꿀 수 있을 것 같아요.🤗